src/components/panel/panel.spec.js

```diff
@@ -1201,6 +1238,34 @@ describe('$mdPanel', function() {
       myButton = angular.element(document.querySelector('button'));
     });
 
+    it('should call onRemoving if provided when removing the panel from the DOM with animation config provided', function() {
+      var onRemoving = jasmine.createSpy();
+      var deferred = $q.defer();
+      var resolvedPromise = deferred.promise;
+
+      var onRemovingAnimation = {
+        'animateClose' : function() {
+          return resolvedPromise;
+        }
+      };
+
+      var animationHookConfig = angular.extend(DEFAULT_CONFIG, {'onRemoving': onRemoving});
```

Comment:
It isn't hard to setup an Animation just use the default fade and you don't need to set the openFrom and closeTo values.